src/benchmark/java/com/commercetools/sync/benchmark/sdk2/BenchmarkUtils.java

```diff
@@ -0,0 +1,125 @@
+package com.commercetools.sync.benchmark.sdk2;
+
+import static java.util.Optional.ofNullable;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import javax.annotation.Nonnull;
+
+final class BenchmarkUtils {
+  private static final String BENCHMARK_RESULTS_FILE_NAME = "benchmarks.json";
+  private static final String BENCHMARK_RESULTS_FILE_DIR =
+      ofNullable(System.getProperty("user.dir"))
+          .map(path -> path + "/tmp_git_dir/benchmarks/")
+          .orElse("");
+  private static final String BENCHMARK_RESULTS_FILE_PATH =
+      BENCHMARK_RESULTS_FILE_DIR + BENCHMARK_RESULTS_FILE_NAME;
+  private static final Charset UTF8_CHARSET = StandardCharsets.UTF_8;
+  private static final String EXECUTION_TIME = "executionTime";
+  private static final String TAG_NAME =
+      ofNullable(System.getenv("GITHUB_TAG")).orElse("DEV-SNAPSHOT");
+
+  static final String PRODUCT_SYNC = "productSync";
```

Comment:
This is never used and will not in future. Comment or delete it maybe?

Reply:
Seems like it's being used to publish the results in the benchmark branch. WDYT?